docs/5.1. Data Model - Identifier Mapping.md

```diff
@@ -4,46 +4,46 @@ _(c) AMWA 2016, CC Attribution-NoDerivatives 4.0 International (CC BY-ND 4.0)_
 
-Sources and Flows must be advertised via a Node's API, and registered via the Registration API if available. This should only be done on the Device which originates the Source or Flow; if a Flow is received by a Device from the network it does not need to be re-advertised via APIs.
+Sources and Flows MUST be advertised via a Node's API, and registered via the Registration API if available. This SHOULD only be done on the Device which originates the Source or Flow; if a Flow is received by a Device from the network it does not need to be re-advertised via APIs.
 
-Whenever a new Flow is generated, its immediate parent Flow(s) must be recorded. This is achieved via a 'parents' attribute in the Node API Flows resource. This contains an array of Flow IDs corresponding to the Flows which have been brought together at this point. For example, an encoder which codes a raw video Flow C into a coded video Flow D would indicate \["C"\] in its 'parents' attribute.
+Whenever a new Flow is generated, its immediate parent Flow(s) MUST be recorded. This is achieved via a `parents` attribute in the Node API Flows resource. This contains an array of Flow IDs corresponding to the Flows which have been brought together at this point. For example, an encoder which codes a raw video Flow C into a coded video Flow D would indicate `["<id of C>"]` in its `parents` attribute.
 
-**Important Note:** A processing element which has a 1:1 input to output relationship and doesn't make editorial changes only need create a new Flow ID on output and not a new Source ID. As a Flow ID must descend from a single consistent Source ID, the Flow ID for these processing elements MUST be generated based on seed data including the incoming Flow ID. If this dynamic Flow generation cannot be achieved, the processing element MUST be viewed as a new Source instead (see Flow ID section below).
+**Important Note:** A processing element which has a 1:1 input to output relationship and doesn't make editorial changes only need create a new Flow ID on output and not a new Source ID. As a Flow ID descends from a single consistent Source ID, the Flow ID for these processing elements MUST be generated based on seed data including the incoming Flow ID. If this dynamic Flow generation cannot be achieved, the processing element MUST be viewed as a new Source instead (see Flow ID section below).
```

Comment:
[As per discussion] `source_id` could match a different Node's Source ID, but the above still stands. It could be written more clearly though:

```suggestion
Where there is a one-to-one input-output relationship with no editorial change, the output only needs to have a new Flow ID, as the same Source ID can be used. In this case the new Flow ID MUST be generated from information that the incoming Flow ID. If this cannot be implemented then a new Source ID MUST be used instead. For more details see the Flow ID section below.
```

Reply:
Looks like there may be a missing word/words in "In this case the new Flow ID MUST be generated from information that the incoming Flow ID"

Reply:
Yes should include "includes". Will fix

Reply:
LGTM